public/style.css

```diff
@@ -19,3 +19,31 @@ form div {
   margin: 1em;
   display: inline-block;
 }
+
+/* jenna's styling */
```

Comment:
Comment is not needed or make description more specific to project's styling

Reply:
Fair enough. I figured this styling is all temporary. Fixed.